### PR TITLE
feat(ArticleCard): add main site article card component

### PR DESCRIPTION
**Pull request by seichang00**
Added article card feature, separate from Article component. Changed some global variables regarding fonts and weights.

### CODE REVIEW

File: src/components/ArticleCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+  type: string
+  title: string
+  content: string
+  imageurl: string
+  photographer: string
+  editor: string
+}
+
+export default class ArticleCard extends React.Component<ArticleCardProps> {
+  public static defaultProps = { imageurl: 'hi' }
```
Comment (zeehang) [inline src/components/ArticleCard/index.tsx]: Should we update this default imageurl?

File: src/components/ArticleCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+              {this.props.photographer}
+            </h3>
+            <h2
+              className={css`
+                color: #1a9ae0;
+                font-size: 0.7em;
+                padding-left: 12px;
+                font-weight: 800;
+                margin-bottom: -20px;
+                font-family: ; //
```
Comment (zeehang) [inline src/components/ArticleCard/index.tsx]: We don't need these extra comment slashes

File: src/index.ts [hunk 3]
```diff
@@ -2,8 +2,9 @@
 /// <reference path="index.d.ts"/>
 
 // Components
-export { default as ArticleGrid } from './components/ArticleGrid'
+// export { default as ArticleGrid } from './components/ArticleGrid'
```
Comment (zeehang) [inline src/index.ts]: If we don't need this we can just take it out

File: src/components/ArticleCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+  imageurl: string
+  photographer: string
+  editor: string
+}
+
+export default class ArticleCard extends React.Component<ArticleCardProps> {
+  public static defaultProps = { imageurl: 'hi' }
+
+  public render() {
+    switch (this.props.type) {
```
Comment (ryang72) [inline src/components/ArticleCard/index.tsx]: the jsx all inside the switch makes it a little hard to read, consider defining each type of card outside and referencing it once in the switch

**Pull request merged**